src/containers/forms/LogActivityForm.jsx

```diff
@@ -19,6 +25,24 @@ class LogActivityForm extends Component {
   */
   static propTypes = {
     categories: PropTypes.arrayOf(PropTypes.shape({})).isRequired,
+    closeModal: PropTypes.func.isRequired,
+    createActivity: PropTypes.func.isRequired,
+  };
+
+  static getDerivedStateFromProps = (nextProps) => {
+    // clear form fields if activity was logged successfully
+    if (nextProps.message && nextProps.message.type === 'success') {
+      return {
+        activityTypeId: '',
```

Comment:
What is the value of using empty string values instead of `null`?

Reply:
I generally use an empty for the data type such that in case I try to perform an operation on that value that is valid for that data type or access a property e.g activityTypeId.length I do not get an error